psalm-baseline.xml

```diff
@@ -265,9 +266,6 @@
       <code><![CDATA[setValue]]></code>
       <code><![CDATA[setValue]]></code>
     </PossiblyNullReference>
-    <PossiblyUndefinedArrayOffset>
-      <code><![CDATA[$newObject['args']]]></code>
-    </PossiblyUndefinedArrayOffset>
```

Comment:
If there is an issue with Psalm, you should report it to Psalm first before using `psalm.xml` to ignore the issue (and when you do, you should put the link to the bug report there). That's why I asked for a reproducer on the other ticket.

Reply:
For This I juste replace the same probleme witch was before in ArrayHydrator and ObjectHydrator to AbstractHydrator.
I don't see ArrayHydrator and ObjectHydrator  in psalm.xml
For the reproducer I'm sorry but I don't understand want you want.

Reply:
Oh yes indeed, those are not really new violations. Should be fine then.